.cirrus.yml

```
@@ -0,0 +1,19 @@
+build_web_task:
```

Seems you can benefit from using [`changesInclude function`](https://cirrus-ci.org/guide/writing-tasks/#supported-functions):

```suggestion
task:
  name: Build Web
  skip: "!changesInclude('.cirrus.yml', 'web/**')"
```

---

Thank you, I didn't know about that!